hosting_docs/source/installation/3-apply-fsck-onreboot.rst

```diff
@@ -0,0 +1,83 @@
+.. _cchq-manual-install:
+
+applying file system check for every reboot 
+====================================================
+
+Checking the file system for errors is an important part of Linux system administration. It is a good troubleshooting 
```

Comment:
Can we  move this doc to firefighting section to here https://github.com/dimagi/commcare-cloud/blob/master/hosting_docs/source/reference/firefighting/general.rst#in-case-of-a-reboot

Reply:
the context is somehow different from the after-reboot. But we can put in https://github.com/dimagi/commcare-cloud/blob/master/hosting_docs/source/reference/firefighting/general.rst